src/qt/locale/bitcoin_ar.ts

```diff
@@ -1211,159 +1197,159 @@
+        <translation>ما إذا كان سيتم إظهار ميزات التحكم في العملة أم لا</translation>
     </message>
     <message>
         <source>Enable coin &amp;control features</source>
-        <translation type="unfinished"></translation>
+        <translation>تمكين ميزات التحكم في العملة</translation>
     </message>
     <message>
         <source>If you disable the spending of unconfirmed change, the change from a transaction cannot be used until that transaction has at least one confirmation. This also affects how your balance is computed.</source>
-        <translation type="unfinished"></translation>
+        <translation>إذا قمت بتعطيل إنفاق التغيير غير المؤكد، فلا يمكن استخدام التغيير من المعاملة حتى تحصل تلك المعاملة على تأكيد واحد على الأقل. ويؤثر هذا أيضًا على كيفية حساب رصيدك</translation>
     </message>
     <message>
         <source>&amp;Spend unconfirmed change</source>
-        <translation type="unfinished"></translation>
+        <translation>قضاء التغيير غير المؤكدة</translation>
     </message>
     <message>
         <source>Automatically open the Dogecoin client port on the router. This only works when your router supports UPnP and it is enabled.</source>
-        <translation type="unfinished"></translation>
+        <translation>فتح منفذ عميل دوجكوين تلقائيًا على جهاز التوجيه. يعمل هذا فقط عندما يدعم جهاز التوجيه الخاص بك UPnP ويتم تمكينه.</translation>
     </message>
     <message>
         <source>Map port using &amp;UPnP</source>
-        <translation type="unfinished"></translation>
+        <translation>منفذ ap باستخدام UPnP</translation>
     </message>
     <message>
         <source>Allow incoming connections</source>
-        <translation type="unfinished"></translation>
+        <translation>السماح بالاتصالات الواردة</translation>
     </message>
     <message>
         <source>Connect to the Dogecoin network through a SOCKS5 proxy.</source>
-        <translation type="unfinished"></translation>
+        <translation>اتصل بشبكة دوجكوين من خلال وكيل SOCKS5</translation>
     </message>
     <message>
         <source>&amp;Connect through SOCKS5 proxy (default proxy):</source>
-        <translation type="unfinished"></translation>
+        <translation>الاتصال عبر وكيل SOCKS5 (الوكيل الافتراضي)</translation>
     </message>
     <message>
         <source>IP address of the proxy (e.g. IPv4: 127.0.0.1 / IPv6: ::1)</source>
-        <translation type="unfinished"></translation>
+        <translation>عنوان IP للوكيل</translation>
     </message>
     <message>
         <source>Shows, if the supplied default SOCKS5 proxy is used to reach peers via this network type.</source>
-        <translation type="unfinished"></translation>
+        <translation>يظهر، إذا تم استخدام وكيل SOCKS5 الافتراضي للوصول إلى النظراء عبر هذا النوع من الشبكة</translation>
     </message>
     <message>
         <source>IPv4</source>
-        <translation type="unfinished"></translation>
+        <translation>IPv4<</translation>
```

Comment:
Extra `<` here, needs to be removed.

Reply:
How can I edit this

Reply:
The bench you created (translate-ar) is linked to this pull request. Just edit the file on that branch and it will update here.

Reply:
Thanks translation completed